packages/neutrino-middleware-env/index.js

```diff
@@ -0,0 +1,5 @@
+const { EnvironmentPlugin } = require('webpack');
+
+module.exports = ({ config }, envs = []) => config
```

Comment:
Is the second argument automatically picked from the "options" in package.json? Or do they all have access to the whole object?